### ISSUE TITLE
"What's a 'production'?"

### ISSUE THREAD

**Issue by Relequestual** [json-schema-org/json-schema-spec#735]
The phrase "A JSON null production" is alien to me.
It reportidly comes from ABNF.
The wikipedia page for EBNF uses the term, but not the RFC related to ABNF that we link to.

Given it doesn't add to the understanding (for me, and others), I discussed with @awwright and he agrees we should remove it.

Looks like this simply means "rule" or the result of following a production rule. We can rephrase this to make it less unintittive for those unfamiliar with computer science terms like this (like myself).

Reference: https://cs.stackexchange.com/q/67417

**Comment by awwright** [comment]
I think I'm the one who invented that phrasing, I think it's the most non-ambiguous way to phrase it, but if there's something more accessible then let's go for it.

I could have sworn ABNF defines the term "production" but it turns out it doesn't, so that's also a good reason to rephrase this.

**Comment by Relequestual** [comment]
I'll make a PR and we'll see what happens =]

**Comment by handrews** [comment]
@Relequestual are you still planning to do anything about this?  Or did it get taken care of somewhere along the way?

**Comment by Relequestual** [comment]
@handrews It did not get taken care of.
I think I now understand why the term was used, but I also think it adds an unnesecary level of required understanding.

In a slack chat, Austin agreed it could be removed in favour of something else, but we didn't discuss what...

Going to keep this open.

**Comment by Relequestual** [comment]
Given that the [JSON website](https://www.json.org/) simply calls these "values", I'm inclined to do the same.

In the other place it's used (https://github.com/json-schema-org/json-schema-spec/blob/38e1606d71ee848c6cf0490c3457681220c01663/jsonschema-core.xml#L234-L235) I'm inclined to simply remove the brackets and contense.

I'll make a PR and we can go from there.
